crates/client-api/src/routes/database.rs

```diff
@@ -798,7 +780,6 @@ pub async fn publish<S: NodeDelegate + ControlStateDelegate>(
 
     // You should not be able to publish to a database that you do not own
     // so, unless you are the owner, this will fail.
-    let auth = auth_or_unauth(auth)?;
```

Comment:
(review note: `create_dns_record` and/or `delete_database` will fail if you're not the right identity anyway)